src/com/google/enterprise/adaptor/sharepoint/SharePointAdaptor.java

```diff
@@ -2223,6 +2227,7 @@ private void getSiteDocContent(Request request, Response response)
         throws IOException {
       log.entering("SiteAdaptor", "getSiteDocContent",
           new Object[] {request, response});
+      Site site = siteDataClient.getContentSite();
```

Comment:
Why get the site this early, instead of leaving it where it was in a smaller scope?

Reply:
shifted site object below